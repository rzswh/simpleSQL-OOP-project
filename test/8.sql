CREATE DATABASE OOP;
USE OOP;
CREATE TABLE poi(lkj INT, mnb INT, bvc INT, dfj CHAR, PRIMARY KEY(mnb));
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (1, 2, 3, "a");
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (2, 3, 4, "a");
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (3, 4, 5, "a");
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (4, 5, 6, "a");
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (5, 6, 7, "a");
UPDATE poi SET dfj="b" WHERE lkj>3;
SELECT dfj from poi;
UPDATE poi SET dfj="c" WHERE lkj>3 OR mnb>2 AND bvc<7 AND lkj=3 OR mnb=2;
SELECT * into outfile "result" from poi;
SELECT * from poi;
DELETE fROm poi where lkj=5;
select lkj,  dfj FROM poi;
DROP DATABASE OOP;
CREATE database OOP2;
USE OOP2; 
CREATE TABLE poi(lkj INT, mnb INT, bvc INT, dfj CHAR, PRIMARY KEY(mnb));
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (1, 2, 3, "a");
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (2, 3, 4, "a");
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (3, 4, 5, "a");
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (4, 5, 6, "a");
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (5, 6, 7, "a");
INSERT INTO poi(mnb, bvc, dfj) VALUES (7, 7, "a");
DELETE FROM poi  WHERE lkj>3 OR 7=mnb AND lkj=10;
SELECT * from poi;
CREATE TABLE poi2(lkj DOUBLE, mnb INT, bvc INT, dfj CHAR, PRIMARY KEY(bvc));
INSERT INTO poi2(lkj, bvc, dfj) VALUES (11, 3, "a");
INSERT INTO poi2 ( lkj , mnb ,bvc ,dfj) VALUES (12, 3, 4, "a")  ;
insert INTO poi2 (lkj, mnb, bvc, dfj) VALUES (13, 4, 5, "a");
INSERT INTO poi2(lkj, mnb, bvc, dfj) VALUES (14, 5, 6, "a");
SELECT * from poi2;
DELETE FROM poi2 WHERE 4.9>mnb;
SELECT * from poi2 where "a"=dfj;
SELECT * from poi2 where NULL>dfj;
SELECT * from poi2 where 'b'>dfj into outfile 'withnull' ;
DROP DATABASE OOP2;
