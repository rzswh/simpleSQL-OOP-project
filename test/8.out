dfj
a
a
a
b
b
lkj     mnb     bvc     dfj
1       2       3       c
2       3       4       a
3       4       5       c
4       5       6       c
5       6       7       c
lkj     dfj
1       c
2       a
3       c
4       c
lkj     mnb     bvc     dfj
1       2       3       a
2       3       4       a
3       4       5       a
NULL    7       7       a
lkj     mnb     bvc     dfj
11.0000 NULL    3       a
14.0000 5       6       a
lkj     mnb     bvc     dfj
11.0000 NULL    3       a
14.0000 5       6       a
lkj     mnb     bvc     dfj
11.0000 NULL    3       a
12.0000 3       4       a
13.0000 4       5       a
14.0000 5       6       a
